helios-testing/src/main/java/com/spotify/helios/testing/Polling.java

```diff
@@ -37,13 +37,14 @@
       }
       Thread.sleep(500);
     }
-    throw new TimeoutException();
+    throw new TimeoutException(String.format(message, timeout, timeUnit.toString().toLowerCase()));
```

Comment:
I figured I could avoid a tiny bit of concatenation at the call site - although really what I was saving was the line on the caller side being even longer. I figured since the only callers are within this same test code that the lack-of-obviousness would be ok